NEWS.rst

```diff
@@ -5,6 +5,60 @@ Release notes
 
 .. towncrier release notes start
 
+Towncrier 23.11.1.dev0 (2024-07-31)
```

Comment:
I think that you have accidentally applied this change ... which was supposed to be only for testing.